src/main/java/frc5124/robot2020/subsystems/Loader.java

```diff
@@ -35,10 +35,13 @@ public void setPower(double power){
     topBeltMotor.set(power);
    // bottomBeltMotor.set(power);
   }
+
+  public double getAppliedOutput() {
+    return topBeltMotor.getAppliedOutput();
+  }
   
   public void runBelt() {
-    topBeltMotor.set(RobotMap.LoaderMap.beltSpeed);
-    //bottomBeltMotor.set(.2);
+    topBeltMotor.set(.420);
```

Comment:
Lets put it back to using the constant, and set the constant to .42 instead of doing it here.
Sorry, this one's my bad.